README.md

```diff
@@ -43,18 +43,7 @@ which will attempt to safely shutdown the virtual machine, or kill it otherwise.
 This approach requires less space, and is faster if your computer already has Python
 and GNU Make installed (i.e. most GNU/Linux machines.)
 
-Developming `hknweb` requires [`pipenv`](https://docs.pipenv.org), which will
-manage the Python virtualenv, selecting the correct Python version, and
-installing all dependencies.
-
-On systems with sudo (root) privileges, you can run
-
-```sh
-$ pip install pipenv
-```
-
-For systems without sudo privileges, the following will work:
-
+Developming `hknweb` requires `venv`.
```

Comment:
Thank you kanye very informative

Pretty sure `venv` module is already included in standard Python 3.

Setup instructions here should go into detail about how to setup a virtualenv and how it works, which is kind of glossed-over currently. https://docs.python.org/3/library/venv.html

Reply:
Added a bit more about why a virtual env is necessary, added link